lib/app.ts

```diff
@@ -37,6 +37,9 @@ export async function run(
 	// shared options, and performs node version requirement checks.
 	globalInit();
 	await routeCliFramework(cliArgs, options);
+
+	// Windows fix: reading from stdin prevents the process from exiting
+	process.stdin.pause();
```

Comment:
Reference: https://github.com/tj/co-prompt/issues/9
By the way, calling `stdin.end()` instead of `stdin.pause()` would cause an error to be thrown.

The reading from `stdin` was taking place at:
https://github.com/balena-io/balena-cli/blob/v11.21.0/lib/utils/remote-build.ts#L106